ui/js/graphs.js

```diff
@@ -748,30 +807,13 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
           function(series) {
             return (series.platform === $scope.selectedPlatform);
           }).sort(function(a, b) { return a.name > b.name; });
-
-        var signatures = new Bloodhound({
-          datumTokenizer: Bloodhound.tokenizers.obj.whitespace('name'),
-          queryTokenizer: Bloodhound.tokenizers.whitespace,
-          limit: 100,
-          local: filteredSeriesList
-        });
-
-        // kicks off the loading/processing of `local` and `prefetch`
-        signatures.initialize();
-
-        if (testInputCreated) {
-          $('.typeahead').typeahead('destroy');
+        testArray = filteredSeriesList;
+        var filteredSeriesName = new Array;
+        for (var keys in filteredSeriesList) {
+            filteredSeriesName.push(filteredSeriesList[keys].name);            
         }
+        $scope.LeftList = filteredSeriesName;
```

Comment:
You could simplify this to something like:

$scope.LeftList = _.map(filteredSeriesList, function(series) { return series.name })

Except you shouldn't use the name `LeftList`. Please rename it to something meaningful like simply `testList`